electron/main.ts

```diff
@@ -10,12 +13,18 @@ moduleAlias.addAliases({
   '@root': `${__dirname}/../`,
 });
 
+import ENV from '../env';
+
+if (!ENV.env || ENV.env === 'development') {
```

Comment:
should the logic perhaps be the other way around - i.e. production is default if no file exists? in which case our builds wouldn't have to be updated?

Reply:
Yes, makes sense. I will change in that way.